test/context.test.ts

```diff
@@ -99,7 +99,26 @@ describe('Context', () => {
     let github: GitHubAPI
 
     function responseFromString (content: string) {
-      return createMockResponse({ content: Buffer.from(content).toString('base64') })
+      const data = {
+        content: Buffer.from(content).toString('base64'),
+
+        // add dummy properties to make response compatible with Typecsript definitions
```

Comment:
If this is just for a test, you can do:

```ts
const data = {  content: '...' } as ActualType
```

Where you would import `ActualType` from `@octokit/rest`.

Reply:
I don't think we have access to `ActualType`, I'm not sure. Can we get it from the `octokit.repos.getContents()` method?